src/Parser/DemonHunter/Vengeance/Modules/Spells/SigilOfFlame.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import Enemies from 'Parser/Core/Modules/Enemies';
+
+import SPELLS from 'common/SPELLS/index';
+import SpellIcon from 'common/SpellIcon';
+
+import { formatPercentage, formatThousands, formatDuration } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class SigilOfFlame extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    enemies: Enemies,
+  };
+
+  previousBuff = 0;
+  currentBuff = 0;
+  difference = 0;
+  stacked = 0;
+
+  on_byPlayer_applydebuff(event){
+    const spellId = event.ability.guid;
+
+    if(spellId !== SPELLS.SIGIL_OF_FLAME_DEBUFF.id) {
+      return;
+    } else{
```

Comment:
Please try to avoid nesting code in additional blocks. Instead you should early return. If you return early, you also won't need `else`, which can be omitted here and further down.